gui/gui-manager.cpp

```diff
@@ -979,7 +979,7 @@ Graphics::MacWindowManager *GuiManager::getWM() {
 		SearchMan.addDirectory(dir);
 	}
 
-	uint32 wmMode = Graphics::kWMModeNoDesktop | Graphics::kWMMode32bpp | Graphics::kWMModeNoCursorOverride;
+	uint32 wmMode = Graphics::kWMModeNoDesktop | Graphics::kWMMode32bpp | Graphics::kWMModeNoCursorOverride | Graphics::kWMModeUnicode;
```

Comment:
Is there a particular reason for this change now?

Reply:
fixed